jenkins-build-pack-api/Jenkinsfile

```diff
@@ -797,10 +796,40 @@ def loadServiceInfo(service, domain, serviceUrl, endpointKey) {
 		preserveEnvKey = 'dev'
 	}
 
-	def curlOutput = sh (script: "curl -H \"Content-Type: application/json\" \
-			-H \"Authorization: $g_login_token\" \"$url\" | $jq -r '.data[0] | [.id, .endpoints[\"$preserveEnvKey\"], .created_by] | join(\",\")'", returnStdout: true)
+	def curlCMDOutput = sh (script: "curl -H \"Content-Type: application/json\" \
+			-H \"Authorization: $g_login_token\" \"$url\"", returnStdout: true)
+			
+	def jsonParser = new groovy.json.JsonSlurper()
+	def resultJson = jsonParser.parseText(curlCMDOutput)
+	def curlOutputJoin = ''
+	if (resultJson != null && resultJson.data != null )
+	{
+		if( resultJson.data[0].id != null){
+			curlOutputJoin = resultJson.data[0].id 
+		}
+		else{
+			curlOutputJoin = curlOutputJoin + "," + ""
+		}
+		
+		if( resultJson.data[0].endpoints != null && resultJson.data[0].endpoints["$preserveEnvKey"] != null){
```

Comment:
check for empty value also.

Reply:
If there is no resultJson.data[0].id this will return null. So I think we need not have to check for empty.

Reply:
what if resultJson.data[0].id is there but == "" ?

Reply:
resultJson.data[0].id cant be an empty value